src/components/related/OutfitList.jsx

```diff
@@ -36,21 +38,26 @@ const OutfitList = () => {
     getStoredOutfit()
   }
 
+
   return (
-    <div className={styles.OutfitList}>
-      Add to Outfit
-      <img
-        src={plusSymbol}
-        style={{ height: '200px', width: '200px'}}
-        onClick={handleAddOutfit}
-      />
-      {Object.values(outfitData).map((storedOutfit) => {
-        return <OutfitCard
-          key={storedOutfit.id}
-          storedOutfit={storedOutfit}
-          getStoredOutfit={getStoredOutfit}
+    <div className='outfit-list'>
+      <div className='add'>
+        <img
+          className='img'
+          src={plusSymbol}
```

Comment:
I like how you moved this out to a variable as well. I need to start doing this more to make my code more readable.